### PR TITLE
AVRO-3251 AVDL files containing fields called "uuid" no longer compile.

### PR DESCRIPTION
**Pull request by martin-g**
### Jira

- [X] My PR addresses the following [AVRO-3251](https://issues.apache.org/jira/browse/AVRO-3251) issue

### Tests

- [X] My PR updates the unit tests

### Commits

- [X] My commits all reference Jira issues in their subject lines.

### Documentation

- [X]  No new functionality. Only bug fixes!

### CODE REVIEW

**Comment by opwvhk** [comment]
Looks good, but incomplete.

There are 31 keywords in the IDL grammar, and only 25 of them are accepted as identifier.
Fields named `idl`, `import`, `oneway`, `null`, `local_timestamp_ms` or `uuid` are all ignored. This commit only fixes the last one.

To verify, see lines 235-264, and lines 1582-1607.

**Comment by martin-g** [comment]
Thanks for the review!

> Fields named `idl`, `import`, `oneway`, `null`, `local_timestamp_ms` or `uuid` are all ignored.

Are you sure about `local_timestamp_ms` ? 
It is at https://github.com/apache/avro/blob/99547e0ee2e0a4a839253848d7b34bac7f5e4cea/lang/java/compiler/src/main/javacc/org/apache/avro/compiler/idl/idl.jj#L263 and https://github.com/apache/avro/blob/99547e0ee2e0a4a839253848d7b34bac7f5e4cea/lang/java/compiler/src/main/javacc/org/apache/avro/compiler/idl/idl.jj#L1605

**Comment by opwvhk** [comment]
Absolutely right Martin! I've done the comparison again, but now with sort&diff instead of by hand... there were only 5 omissions (including uuid), instead of 6.

File: lang/java/compiler/src/test/idl/input/reservedwords.avdl [hunk 1]
```diff
@@ -19,4 +19,9 @@
 protocol Foo {
   void `error`();
   void `void`();
+  void `idl`();
+  void `import`();
+  void `oneway`();
+  void `null`();
+  void `local_timestamp_ms`();
```
Comment (opwvhk) [inline lang/java/compiler/src/test/idl/input/reservedwords.avdl:26]: 👍

**Review by opwvhk — APPROVED** [review]
LGTM

**Review by RyanSkraba — APPROVED** [review]

**Pull request merged**